frontend/workflows-lib/tests/components/TasksDynamic.test.tsx

```diff
@@ -1,31 +1,31 @@
 import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
```

Comment:
is this import still needed?

Reply:
Yeah, weirdly `jest-dom` isn't actually `jest` specific and is recommended as the to test the dom by `testling-library`, see: https://testing-library.com/docs/svelte-testing-library/setup/#vitest